lib/foreman_maintain/concerns/system_helpers.rb

```diff
@@ -184,18 +190,44 @@ def get_lv_path(lv_name)
         execute("lvs --noheadings -o lv_path -S lv_name=#{lv_name}").strip
       end
 
-      def local_psql_database?
-        (feature(:foreman_database) && feature(:foreman_database).local?) ||
-          (feature(:candlepin_database) && feature(:candlepin_database).local?)
-      end
-
       def find_dir_containing_file(directory, target)
         result = nil
         Find.find(directory) do |path|
           result = File.dirname(path) if File.basename(path) == target
         end
         result
       end
+
+      def foreman_proxy_with_content?
```

Comment:
I'd suggest move all these to `feature(:instance)` See comment above for reasons